shared/naturalcrit/markdown.js

```diff
@@ -127,7 +127,7 @@ const mustacheSpans = {
 	start(src) { return src.match(/{{[^{]/)?.index; },  // Hint to Marked.js to stop and check for a match
 	tokenizer(src, tokens) {
 		const completeSpan = /^{{[^\n]*}}/;               // Regex for the complete token
-		const inlineRegex = /{{(?=((?:[:=](?:"['\w,\-()#%=?.: ]*"|[\w\-()#%.]*)|[^"=':{}\s]*)*))\1 *|}}/g;
+		const inlineRegex = /{{(?=((?:[:=](?:"['\w,\-()#%=?.: ]*"|'[\w,\-()#%=?.: ]*'|[\w\-()#%.]*)|[^"=':{}\s]*)*))\1 *|}}/g;
```

Comment:
I'm not sure I understand why we needed to allow `'single quotes'` for this PR to work? If not needed can we remove this change?

Reply:
I suppose you can.

Reply:
Reverted in next commit.